pkg/operator/observe_config.go

```diff
@@ -149,6 +150,63 @@ func observeInternalRegistryHostname(kubeClient kubernetes.Interface, clientConf
 	return observedConfig, nil
 }
 
+func observeBuildControllerConfig(kubeClient kubernetes.Interface, clientConfig *rest.Config, observedConfig map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
You'll need to add this function to the list of observeConfigFuncs in the constructor